content/12.raw-data-analysis.md

```diff
@@ -61,6 +62,27 @@ The existing software for DDA and DIA analysis can be further divided into freew
 
 
 ### Analysis of DDA data
+DDA data analysis can be performed in one of the two ways- either with the vendor propreitary format directly analyzed using a propreitary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example - MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, OMSSA, or Trans-Proteomics pipeline (TPP).
+The raw data is converted to either text-based MGF (mascot generic format) or into stadard open XML implementation like mzML format.
+The appropriate FASTA file is chosen as a reference database to search the experimental spectra.
+All search prarameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artefacts (fixed modifications) and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
+The search algorithm fetches the best possible match for each query spectrum, from among a list of scored candidate peptides for that query spectrum. Each spectrum and its corresponding scored candidate peptide is called a peptide spectrum match (PSM).
+The scores reflect a _goodness-of-fit_ between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.
+For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
+A randomized or reversed version of target database is used as a nonparametric null model.
+The decoy database can be searched separate from the target database (Kall's method)[@DOI:10.1021/pr700600n] or it can be combined with the target database before search (Elias and Gygi method)[@DOI:10.1038/nmeth1019].
+Using either separate method or concatenated database search method, an estimate of false hits can be calculated which is used to estimate the false discovery rate (FDR) [@DOI:10.1007/978-1-4939-3106-4_7]. 
+The FDR denotes the proportion of false hits in the population accepted as true.
+For Kall's method: the false hits are estimated to be the number of decoys above a given threshold. It is assumed that the number of decoy hits that pass a threshold are the false hits. A similar number of target population may also be false. Therefore, the FDR is calculated as-
+FDR = #Decoy PSMs/#Target PSMs
+
+For Elias and Gygi Method, the target population in which FDR is estimated changes.
+The target and decoy hits coming from a joint database compete against each other.
+For any spectrum, either a target or a decoy peptide can be the best hit.
+It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
+However, due to the joint database search, the target database may also have equal number of false hits. Thus, the number of false hits is doubled during FDR estimation.
+FDR = (2 * # Decoy PSMs)/ (#(Target+Decoy) PSMs)
```

Comment:
please convert to latex style equation - see here for example: https://github.com/agitter/my-manuscript/blob/452ddcebf15b183dd3dec9cb5f89af027cc4daa3/content/03.delete-me.md#equations

and here:

https://www1.cmc.edu/pages/faculty/aaksoy/latex/latexthree.html